### DISCUSSION TITLE
What determines the late submission

### DISCUSSION THREAD

**Discussion by ghost** [idena-network/idena-go#638] [Q&A] [answered]
Hello i would like to know what determines the late submission technically 🤔

**Comment by midenaio** [comment, marked as answer]
Every node votes with 1 or 0 about each address by sending the `EvidenceMapTx` transaction.
`1` - valid submission
`0` - late submission
50% of `1` votes are required to determine if the submission status is valid. 

A node votes for `1` if both of these 2 conditions are met:
1. Short answers hash message was seen during the short session
2. Flip keys (needed to decrypt author's flips) were seen during the first 30 seconds of the short session.
